common/config/pinot.go

```diff
@@ -23,9 +23,14 @@ package config
 // PinotVisibilityConfig for connecting to Pinot
 type (
 	PinotVisibilityConfig struct {
-		Cluster     string `yaml:"cluster"`     //nolint:govet
-		Broker      string `yaml:"broker"`      //nolint:govet
-		Table       string `yaml:"table"`       //nolint:govet
-		ServiceName string `yaml:"serviceName"` //nolint:govet
+		Cluster     string         `yaml:"cluster"`     //nolint:govet
+		Broker      string         `yaml:"broker"`      //nolint:govet
+		Table       string         `yaml:"table"`       //nolint:govet
+		ServiceName string         `yaml:"serviceName"` //nolint:govet
+		Migration   PinotMigration `yaml:"migration"`   //nolint:govet
+	}
+
+	PinotMigration struct {
```

Comment:
Yeah totally agree! I have been thinking how to refactor the triple visibility manager with primary/source datastore and secondary/target datastore, then this config should also be updated then. I think it will happen in H2 or when we need to migrate the remaining ES clusters to Open search.